client/modules/users/authReducer.js

```diff
@@ -91,6 +132,18 @@ export const loadUser = () => ({
   }
 })
 
+const rmNotification = (state, actionIndex) => {
+  const newState = state
```

Comment:
This will mutate the state, the state var is a reference to the state, so in this line you assign the same reference to newState. So both state and newState reference the same data and changing one will change the other. So you have to do something like this:
```
const notifications = [
  ...state.user.notifications.slice(0, index),
  ...state.user.notifications.slice(index + 1),
]

return {
  ...state,
  user: {
    ...state.user,
    notifications
  }
}
```